content/07.rule6.md

```diff
@@ -0,0 +1 @@
+## Rule 6: Limit overfitting
```

Comment:
Alt. suggestion:

"Address deep neural networks' increased tendency to overfit the dataset"

- also here, to make this a bit more DL specific, since DL models, through overparameterization, tend to overfit more than off-the-shelf algorithms like random forests etc.